app/Payout.hs

```diff
@@ -122,15 +122,17 @@ payout (Config baker host port from fromName varyingFee databasePath accountData
         return (res, (updated, return ()))
 
       maybePayoutDelegatorsForCycle cycle delegators db = do
-        let needToPay = P.filter (\(_, delegator) -> case (delegatorPayoutOperationHash delegator, delegatorFinalRewards delegator) of (Nothing, Just amount) | amount > 0 -> True; _ -> False) $ M.toList delegators
+        let needToPay = if payEstimatedRewards then P.filter (\(_, delegator) -> case delegatorPayoutOperationHash delegator of Nothing -> True; _ -> False) $ M.toList delegators else P.filter (\(_, delegator) -> case (delegatorPayoutOperationHash delegator, delegatorFinalRewards delegator) of (Nothing, Just amount) | amount > 0 -> True; _ -> False) $ M.toList delegators
```

Comment:
Estimated rewards could still be `0`, I believe - do you want to send a transaction in that case?

Reply:
No. I just pushed a change that makes this right.